packages/api/src/controllers/webhook.ts

```diff
@@ -237,6 +234,39 @@ app.patch(
   }
 );
 
+app.post(
+  "/:id/status",
+  authorizer({ anyAdmin: true }),
+  validatePost("webhook-status-payload"),
+  async (req, res) => {
+    const { id } = req.params;
+    const webhook = await db.webhook.get(id);
+    if (!webhook) {
+      return res.status(404).json({ errors: ["not found"] });
+    }
+
+    const doc = req.body.status;
+    const webhookResponse = req.body.response;
+
+    if (!doc) {
+      return res.status(422).json({ errors: ["missing status in payload"] });
+    }
+
+    await db.webhook.update(req.params.id, {
```

Comment:
There is a dedicated `updateStatus` function which should be used instead. Try taking a look on the `cannon.ts` code that does these updates to make sure you are doing the same thing here (or if there's anything you could reuse, like exposing some of the logic there as helpers):
https://github.com/livepeer/livepeer-com/blob/0f808b4b87eb72942eb098e8c3e4f13ea7699a90/packages/api/src/webhooks/cannon.ts#L446-L503